Source/CarthageKit/Netrc.swift

```diff
@@ -0,0 +1,60 @@
+import Foundation
+
+internal struct NetrcMachine {
+    let name: String
+    let login: String
+    let password: String
+}
+
+internal struct Netrc {
+    
+    enum NetrcError: Error {
+        case fileNotFound(String)
+        case unreadableFile(String)
+        case machineNotFound
+        case missingToken(String)
+        case missingValueForToken(String)
+    }
+    
+    static func load(from file: URL = URL(fileURLWithPath: "\(NSHomeDirectory())/.netrc")) throws -> [NetrcMachine] {
```

Comment:
Please use Result instead.  Do not throw.

Reply:
Thanks for an advice, I replaced throwing errors with `Result` type 🙇